MAUI/DataGrid/column-resizing.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Column Resizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column reizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
```

Comment:
Learn all about column resizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.

Reply:
Addressed the review concerns provided.